src/main/java/nextstep/subway/ErrorResponse.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.subway;
+
+public class ErrorResponse {
+  private final String message;
+
+  public ErrorResponse(String message) {
+    this.message = message;
+  }
+
+  public static ErrorResponse of(String message) {
+    return new ErrorResponse( message);
```

Comment:
```suggestion
    return new ErrorResponse(message);
```

맥 + 인텔리제이를 사용하고계시면 항상 커밋 전 파일에 대해 `code prettier` 를 적용하면 좋을 것 같습니다.
단축기: `option + command + L`

Reply:
아하 넵 팁 감사합니다!